### PR TITLE
add a bit to the examples/real

### PR DESCRIPTION
**Pull request by adamtopaz**

### CODE REVIEW

File: src/examples/real.lean [hunk 1]
```diff
@@ -14,8 +19,14 @@ open liquid_tensor_experiment category_theory category_theory.limits opposite
...
 
--- TODO: once we bump mathlib past [#3292](https://github.com/leanprover-community/mathlib/pull/3292) we can show that `ℝ` is the unique
--- conditionally complete linearly ordered field, up to isomorphism.
+-- Any conditionally complete linear ordered field is isomorphic (as an ordered ring) to `ℝ`.
+example {R : Type*} [conditionally_complete_linear_ordered_field R] : R ≃+*o ℝ := default
+
+-- The isomorphism above is unique
+example {R : Type*} [conditionally_complete_linear_ordered_field R] (e₁ e₂ : R ≃+*o ℝ) :
+  e₁ = e₂ :=
+(eq_iff_true_of_subsingleton _ _).mpr trivial
```
Comment (jcommelin) [inline src/examples/real.lean:32]: Is this not just `subsingleton.elim e₁ e₂`?

File: src/examples/real.lean [hunk 2]
```diff
@@ -14,8 +19,14 @@ open liquid_tensor_experiment category_theory category_theory.limits opposite
 
 example : conditionally_complete_linear_order ℝ := infer_instance
 example : linear_ordered_field ℝ := infer_instance
+example : conditionally_complete_linear_ordered_field ℝ := infer_instance
```
Comment (jcommelin) [inline src/examples/real.lean:22]: I think we can just remove the two lines above this.

File: src/examples/real.lean (hunk 1 above)
Comment (adamtopaz) [reply src/examples/real.lean:32]: This was the handy work of library search. I changed it ;)

File: src/examples/real.lean (hunk 2 above)
Comment (adamtopaz) [reply src/examples/real.lean]: Done

File: src/examples/real.lean [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
 example : ℝ≥0 = {r : ℝ // r ≥ 0} := rfl
 
--- TODO: once we bump mathlib past [#3292](https://github.com/leanprover-community/mathlib/pull/3292) we can show that `ℝ` is the unique
--- conditionally complete linearly ordered field, up to isomorphism.
+-- Any conditionally complete linear ordered field is isomorphic (as an ordered ring) to `ℝ`.
+example {R : Type*} [conditionally_complete_linear_ordered_field R] : R ≃+*o ℝ := default
+
+-- The isomorphism above is unique
+example {R : Type*} [conditionally_complete_linear_ordered_field R] (e₁ e₂ : R ≃+*o ℝ) :
+  e₁ = e₂ := subsingleton.elim _ _
```
Comment (jcommelin) [inline src/examples/real.lean:29]: We could potentially just phrase this as an instance of `unique (R ≃+*o ℝ)`, now that I think of it.

**Review by jcommelin — APPROVED** [review]

**Pull request merged**